packages/components/releaseNotes/components.md

```diff
@@ -5,6 +5,10 @@ Components, models, actions, and utility functions for LabKey applications and p
 *Released*: 1 November 2021
 * Previous release was built against an incorrect version of api-js which broke workflow task support included in 2.87.0
 
+### version TBD
+*Released*: TBD
+* Ensure LK instances without LKSM do not call prefix-related actions
```

Comment:
looks like this update got put below the v2.90.1 note above.

Reply:
Oop, moved!